### PR TITLE
Forbids drinking and eating food/pills while wearing mouth-covering masks/helmets

### PR DESCRIPTION
**Pull request by unid15**
EDIT: **THIS FEATURE IS NOW DISABLED BY DEFAULT, AND CAN ONLY BE ACTIVATED BY AN ADMIN**

Golems still can eat
Mime / clown masks don't forbid anything so you can eat without having to remove them

Reminder that syringes can still be used to administer medicine while in space

Other changes:
-Added a "hardcore" variable to config and a "toggle hardcore" command for funmins. Turning hardcore mode on will enable this feature
-Removed duplicate definition of golem's mask in the file, scarves now extend the /mask/scarf subtype, fake mustache doesn't cover any part of the face anymore and muzzles only cover the mouth

### CODE REVIEW

**Comment by Iamgoofball** [comment]
This is one of the most hated changes paprika made at tg, be warned.

**Comment by FudgePucker** [comment]
Fuck
No

**Comment by Unfit** [comment]
muh realisms

**Comment by SkowronX** [comment]
Man that's gonna suck for plasmamen.

**Comment by FudgePucker** [comment]
Engineers already have it hard enough, now they are going to have to fucking juggle three objects just to eat?
Fuck
No
This is the opposite of a 'quality of life' improvement.

**Comment by PJB3005** [comment]
THINK OF ALL THE PLASMAMEN AND VOX

**Comment by NigglyWiggly** [comment]
>Vox can no longer eat anything without breathing toxic air

This is a good thing.

File: code/modules/admin/admin_verbs.dm [hunk 1]
```diff
@@ -1014,4 +1015,29 @@ var/list/admin_verbs_mod = list(
...
+	set category = "Fun"
+
+	if(!holder || !config)
+		return
+
+	if(!config.hardcore)
+		usr << "<font color='blue'>*----------------------*</font>"
+		usr << "Currently supported hardcore features:<br>"
+		usr << "- Eating, drinking or taking pills while wearing a mouth-covering helmet or mask is <font color='red'>impossible</font>"
+		usr << "<font color='blue'>*----------------------*</font>"
```
Comment (PJB3005) [inline code/modules/admin/admin_verbs.dm]: `<span style="color: blue;"></span>`

**Comment by SangriaMonkey** [comment]
Don't do this.

**Comment by NigglyWiggly** [comment]
Do this.

**Comment by d3athrow** [comment]
There are very, very few servers that use our codebase downstream and this will not fly at all on /vg/station. 

Main reasons being 2 of the non human races need to have their helmet and/or mask on or they will die.

**Comment by NigglyWiggly** [comment]
Pomf confirmed nofun

**Comment by FudgePucker** [comment]
Reasons why we are better than /tg/, number 1#: pomf is our host and is a pretty cool guy.

**Pull request closed without merging**